utils/rpms/daos.spec

```diff
@@ -250,7 +250,8 @@ getent passwd daos_server >/dev/null || useradd -M daos_server
 %dir %{_prefix}%{_sysconfdir}
 %{_prefix}%{_sysconfdir}/vos_dfs_sample.yaml
 %{_prefix}%{_sysconfdir}/vos_size_input.yaml
-%{_libdir}/libdaos_common.so
+%{_libdir}/daos_srv/libdaos_common.so
+%{_libdir}/daos_srv/libdaos_common_cli.so
```

Comment:
This seems wrong.   I agree that the server should be the one we rename..it will be friendlier to users if they don't have a library name change.

Regardless, the server one should only be installed with the server and the client one should only be installed on the client

Reply:
Ah, perhaps it's due to vos_size tool.  Next step would be to remove the PMDK specific stuff from that tool :-)

Reply:
Ok then I will change the server common library name.

Reply:
Fixed in latest patch.